app/assets/stylesheets/application.scss

```diff
@@ -27,6 +27,18 @@ header .gem-c-metadata {
   header {
     background: $govuk-blue;
 
+    // FIXME this is to fix link colours in the metadata component on a
+    // blue background. This can be removed once the component has been
+    // updated to accommodate such a situation
+    .gem-c-metadata a,
+    .gem-c-metadata a:link,
+    .gem-c-metadata a:hover,
```

Comment:
Do we need to maintain a distinct hover state colour, or is it OK to have them all be white like this?

Reply:
I did wonder about that, but I don't know what the hover colour should be - this has to work on a dark blue background and also a teal-ish background. Changing link colour on hover isn't an accessibility requirement (I think) so I think for now it's easier if we just leave them white.